packages/beacon-node/src/monitoring/system.ts

```diff
@@ -5,264 +5,211 @@ import {Logger} from "@lodestar/utils";
-  diskNodeBytesFree = fileSystem.available;
-
-  if (diskIOMeasurable) {
-    const disk = await system.disksIO();
-    if (disk != null && disk.rIO !== 0) {
-      // Note: rIO and wIO might not be available inside container
-      // see https://github.com/sebhildebrandt/systeminformation/issues/777
-      diskNodeReadsTotal = disk.rIO;
-      diskNodeWritesTotal = disk.wIO;
-    } else {
-      diskIOMeasurable = false;
+  private async collectDiskData(): Promise<void> {
+    const fileSystems = await system.fsSize();
+    // get file system root, on windows this is the name of the hard disk partition
+    const rootFs = process.platform === "win32" ? process.cwd().split(path.sep)[0] : "/";
+    // only consider root file system, if it does not exist use first entry in the list
+    const fileSystem = fileSystems.find((fs) => fs.mount === rootFs) ?? fileSystems[0];
+    this._diskNodeBytesTotal = fileSystem.size;
+    this._diskNodeBytesFree = fileSystem.available;
+
+    if (this.diskIOMeasurable) {
+      const disk = await system.disksIO();
+      if (disk != null && disk.rIO !== 0) {
+        // Note: rIO and wIO might not be available inside container
+        // see https://github.com/sebhildebrandt/systeminformation/issues/777
+        this._diskNodeReadsTotal = disk.rIO;
+        this._diskNodeWritesTotal = disk.wIO;
+      } else {
+        this.diskIOMeasurable = false;
+      }
     }
   }
-}
 
-async function collectNetworkData(): Promise<void> {
-  // defaults to first external network interface
-  const [network] = await system.networkStats();
-  // Note: rx_bytes and tx_bytes will be inaccurate if process
-  // runs inside container as it only captures local network traffic
-  networkNodeBytesTotalReceive = network.rx_bytes;
-  networkNodeBytesTotalTransmit = network.tx_bytes;
-}
+  private async collectNetworkData(): Promise<void> {
+    // defaults to first external network interface
+    const [network] = await system.networkStats();
+    // Note: rx_bytes and tx_bytes will be inaccurate if process
+    // runs inside container as it only captures local network traffic
+    this._networkNodeBytesTotalReceive = network.rx_bytes;
+    this._networkNodeBytesTotalTransmit = network.tx_bytes;
+  }
 
-function getNormalizedOsVersion(): MiscOs {
-  switch (process.platform) {
-    case "linux":
-      return "lin";
-    case "darwin":
-      return "mac";
-    case "win32":
-      return "win";
-    default:
-      return "unk";
+  private getNormalizedOsVersion(): MiscOs {
+    switch (process.platform) {
+      case "linux":
+        return "lin";
+      case "darwin":
+        return "mac";
+      case "win32":
+        return "win";
+      default:
+        return "unk";
+    }
   }
-}
 
-function getSystemBootTime(): number {
-  return Math.floor(Date.now() / 1000 - os.uptime());
-}
+  private getSystemBootTime(): number {
+    return Math.floor(Date.now() / 1000 - os.uptime());
+  }
 
-/**
- * Number of CPU cores available
```

Comment:
I removed all of the function docs as those are already documented in `clientStats.ts` and I think it is better not to duplicate this, also not sure how much value the docs provide here so I went with reducing duplication and verbosity

https://github.com/ChainSafe/lodestar/blob/b64a67c71513e5d8b0ff2b8479ee90ed84da94d4/packages/beacon-node/src/monitoring/clientStats.ts#L201-L205